bench.css

```
@@ -0,0 +1,687 @@
+  box-sizing: border-box;
+}
+
+:root {
+  --bg: #0a0a0a;
+  --surface: #111;
+  --border: #1c1c1c;
+  --text: #ededed;
+  --text-dim: #888;
+  --text-muted: #444;
+  --accent: #D97757;
+  --serif: 'DM Serif Display', Georgia, serif;
```

Stylelint (17.3.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)